final_task/rss_reader/classes/news_class.py

```diff
@@ -22,17 +22,34 @@ def print_feed_title(self):
         """Print feed title"""
         print(self.feed_title)
 
+    def create_list_of_links(self):
+        """Turn tuple ([href_links], [img_links]) into list of links"""
+        # Merge list of links
+        list_of_links = []
+        list_of_links.extend(self.links[0])
+        list_of_links.extend(self.links[1])
+        return list_of_links
+
+    def create_string_of_links(self):
+        """Turn tuple ([href_links], [img_links]) of lists into formatted string"""
+        # Merge list of links
+        list_of_links = self.create_list_of_links()
+
+        string_repr_of_links = ''
+        for num, link in enumerate(list_of_links):
+            if link:
+                string_repr_of_links = string_repr_of_links + '[{}] '.format(num + 1) + link + '\n'
+        self.links = string_repr_of_links
```

Comment:
It is better to not change existing object, but to return new object. This change is too implicit, and someone unfamiliar with your code will wonder why at some point of app work `self.links` has turned to a string.

Reply:
Fully agreed

Reply:
Fixed